src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -220,18 +219,17 @@ internal static void FreeSslContext(SafeSslHandle context)
         #endregion
 
         #region private methods
-        private static IntPtr GetSslMethod(bool isServer, long options)
+        private static IntPtr GetSslMethod(SslProtocols protocols)
```

Comment:
We deliberately didn't use SslProtocols because that introduces a dependency on System.Net.Security. Interop.OpenSsl.cs is meant to be xplat-only so that other libraries like System.Net.HttpClient can use it easily. Filed issue# 4129